frontend/simple/sass/_groupincome.scss

```diff
@@ -1,123 +1,27 @@
-.centered {
-  margin: 0 auto;
-}
-
-.percentage {
-  height: 26px;
-  color: inherit;
-  border: none;
-  outline: none;
-  font-size: 28px;
-}
-
-@mixin appearance($value) {
-  -webkit-appearance: $value;
-  -moz-appearance: $value;
-  -ms-appearance: $value;
-}
-
-@mixin transition($value) {
-  -webkit-transition: $value;
-  -moz-transition: $value;
-  -ms-transition: $value;
-  transition: $value;
-}
-
-@mixin transform($value) {
-  -webkit-transform: $value;
-  -moz-transform: $value;
-  -ms-transform: $value;
-  transform: $value;
-}
-
-/* Slider toggle Styles */
-
-/* The switch - the box around the slider */
-.switch {
-  position: relative;
-  display: inline-block;
-  width: 60px;
-  height: 34px;
-
-  input {
-    display: none;
-  }
-}
-
-.text-message {
-  margin: $size-5 0;
-
-  &.is-success {
-    color: $green;
-  }
-}
-
-.create-group {
-  .select::after {
-    border-color: #ccc;
-  }
-
-  .box {
-    margin-bottom: 0;
-    border-radius: 0;
-  }
-}
-
-.centered-button {
-  max-width: 50%;
-}
-
 @font-face {
   font-family: Foco;
   src: url(../fonts/FocoBold.ttf);
 }
 
-.foco-font {
-  font-family: Foco;
-}
+// ------ Brand colors ------ //
 
-.unread {
-  display: inline-block;
-  padding: 2px 5px;
-  font-size: 12px;
-  font-weight: 600;
-  line-height: 1;
-  color: white;
-  background-color: #1fc8db;
-  border-radius: 20px;
-}
-
-//--------------------------------------
-// Modifiers
-//--------------------------------------
-.is-danger {
-  color: $red;
-}
-
-.is-success {
-  color: $green;
-}
+$gi-brand-orange: #f89201;
+$gi-brand-green: #a0d10e;
+$gi-brand-blue: #5dc8f0;
+$gi-brand-blue-dark: #1fc8db;
 
-.is-warning {
-  color: $orange;
-}
-
-.is-light {
-  color: $grey;
-}
-
-.is-small {
-  font-size: $size-7;
-}
-
-// Colors variants matching the background
+// ------ Utilities ------ //
+.gi- {
```

Comment:
👍 so now this is global and theme related, so I can totally accept the gi-prefix in this case.

Reply:
Cool!